shared/test/test_assets.rb

```diff
@@ -186,10 +186,10 @@ def test_viewing_abusive_assets
     FilesApi.any_instance.unstub(:admin?)
 
     # teacher can view
-    FilesApi.any_instance.stubs(:teaces_student?).returns(true)
+    FilesApi.any_instance.stubs(:teaches_student?).returns(true)
```

Comment:
I would be a bit worried about whether this test is actually testing what it says it is testing if it works both ways.. do you understand why that is?